src/app/components/event-card.tsx

```diff
@@ -26,7 +26,7 @@ export function EventCard(props: EventCardProps): React.JSX.Element {
           <styled.span textStyle="label.01">{title}</styled.span>
           {/* TODO: What does this do? */}
           {actions && (
-            <styled.button width="24px" position="absolute" right={0}>
+            <styled.button width="24px" position="absolute" right={0} type="button">
```

Comment:
Did we ever find out what this does? Maybe we can remove the `TODO: What does this do?` above?

Reply:
I'll double check this today.